packages/xo-web/src/common/xo-compression/index.js

```diff
@@ -0,0 +1,8 @@
+export SelectCompression from './select-compression'
+
+// zstd detection source: https://github.com/xapi-project/xen-api/commit/adca3b750da06deb9a141ad2447605601c49a177
```

Comment:
IMHO, add link to code source or stable branch but not to a commit link.

Reply:
Reason?

Reply:
It's my fault because it's the same.